ui/src/components/MotorInput/OneAxisTranslationControl.module.css

```diff
@@ -0,0 +1,21 @@
+.input {
+  background-color: var(--hw-ready--bg) !important;
+  transition: background-color 100ms ease-in;
```

Comment:
Each component redefines its own state-based styles (but using the global CSS custom properties). This gives more flexibility (which HWO states to style, which styles to apply, etc.)